ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -86,6 +86,24 @@ describe("Form with File Input - Basic Tests", () => {
                 checkHTML(id, field.getModel().getState())
         });
     })
+    it(" clicking fileinput after attaching file resets the widget value ", () => {
```

Comment:
The commit message and the test does not match. Can you elaborate on the use-case ?